CHANGELOG.md

```diff
@@ -149,6 +149,7 @@
 - Bugfix: Fixed crash happening when QuickSwitcher is used with a popout window. (#4187)
 - Bugfix: Fixed low contrast of text in settings tooltips. (#4188)
 - Bugfix: Fixed being unable to see the usercard of VIPs who have Asian language display names. (#4174)
+- Bugfix: Fixed whispers always being shows in the /mentions split. (#4389)
```

Comment:
```
- being shows
+ being shown
```

Reply:
thank you 
![image](https://user-images.githubusercontent.com/962989/219863193-5c66db7d-bca8-496b-ab6c-7f8593c0e662.png)